bindings/python/examples/client/submit_and_read_block.py

```diff
@@ -80,9 +80,9 @@
 
 # Get the whole block
 block = client.get_block_data(block_id)
-payload_out = block['payload']
-tag_hex_out = block['payload']['tag']
-message_hex_out = block['payload']['data']
+payload_out = block.payload
+tag_hex_out = block.payload['tag']
+message_hex_out = block.payload['data']
```

Comment:
rename to `data_hex_out`?

Reply:
But it's about the message defined in L34

Reply:
I have to work with the context Gh is providing me! :trollface: